cartofacade/capi.go

```diff
@@ -79,12 +77,6 @@ type Position struct {
 	ComponentReference string
 }
 
-// IMUReading holds values for linear acceleration and angular velocity to be converted into c
-type IMUReading struct {
-	LinearAcceleration r3.Vector
-	AngularVelocity    spatialmath.AngularVelocity
-}
```

Comment:
This struct's information is already contained in `s.TimedIMUReadingResponse` and can be removed & replaced.